dali/pipeline/util/bounding_box.h

```diff
@@ -49,24 +62,27 @@ class BoundingBox {
     swap(lhs.area_, rhs.area_);
   }
 
-  static BoundingBox FromLtrb(const float* data) {
-    return FromLtrb(data[0], data[1], data[2], data[3]);
+  static BoundingBox FromLtrb(const float* data, float4 bounds = UniformSquare()) {
```

Comment:
bounds should be std::array<float, kSize>

Reply:
Fixed